### DISCUSSION TITLE
cannot read property 'start' of null

### DISCUSSION THREAD

**Discussion by Dhurika** [innoveit/react-native-ble-manager#1193] [Q&A]
ERROR  BeManager could not be started. [TypeError: Cannot read property 'start' of null]
 ERROR  [retrieveConnected] unable to retrieve connected peripherals. [TypeError: Cannot read property 'getConnectedPeripherals' of null]
 DEBUG  [startScan] starting scan...
 ERROR  [startScan] ble scan returned in error [TypeError: Cannot read property 'scan' of null]
 ERROR  [retrieveConnected] unable to retrieve connected peripherals. [TypeError: Cannot read property 'getConnectedPeripherals' of null]
Actions
how do i resolve and solve it

**Comment by salmantln** [comment]
Did you manage to fix this?
